src/components/AppHeader/MobileMenu/SubmenuContent.tsx

```diff
@@ -0,0 +1,151 @@
+import { BackButton } from './BackButton';
+import { MobileMenuConfig, TSubmenuSection } from './MobileMenuConfig';
+
+type TSubmenuContent = {
+    onBackClick: () => void;
+    renderContentFor: TSubmenuSection;
+};
+
+export const SubmenuContent = ({ onBackClick, renderContentFor }: TSubmenuContent) => {
+    const data = MobileMenuConfig().submenuConfig[renderContentFor];
+
+    return (
+        <div className='pb-[32px]'>
+            <BackButton buttonText={data.title} onClick={onBackClick} />
+
+            {/* <div className='flex items-center pt-[20px] pl-[32px]'>
```

Comment:
is there a plan for this? if not necessary, let's remove it.

Reply:
As I mentioned it's inProgress I'll update this in another PR